src/utils/session-helper.js

```diff
@@ -289,8 +293,12 @@ const createFileChecksum = async (fileObject) => {
  * @param {fileObjectType} fileObject - Object containing information about file
  * from form submission (see {@link fileObjectType})
  * @param {string} documentUrl - url of uploaded document or resource
+<<<<<<< HEAD
```

Comment:
Found lines related to merge conflicts from the recent rebasing.

Reply:
fixed